lib/redis.js

```diff
@@ -0,0 +1,14 @@
+const Redis = require("ioredis");
```

Comment:
Just curious, why did you pick ioredis instead of https://www.npmjs.com/package/redis?